wpeverest/user-registration
# Enhance - Valid file type in my account profile picture

### All Submissions:


### Changes proposed in this Pull Request:
Previously, when the users wants to change the profile picture there is no method to check the valid file types for profile picture. This PR provides an option to choose the valid file type so the users submit only allowed files.

### How to test the changes in this Pull Request:

1.Navigate to the form builder and drag and drop the Profile Picture
2.In the Advance setting of field options of profile picture field there will be option to choose the valid file types.
3.Navigate to the registration form and verify whether the valid file are allowed or not.
4. Also check in the edit-profile and verify whether the valid file are allowed or not .

### Dependent PR:
https://github.com/wpeverest/user-registration-advanced-fields/pull/22

### Types of changes:


### Other information:


### Changelog entry

>Enhance - Valid file type in my account profile picture
